src/multiselect/__integ__/multiselect.test.ts

```diff
@@ -108,7 +108,7 @@ const createSetupTest = (wrapper: MultiselectWrapper) => (testFn: (page: Multise
 });
 
 describe('focus handling', () => {
-  test('focuses trigger after dismissing a token', () => {
+  test('focuses trigger after dismissing all tokens', () => {
```

Comment:
do we really want to change this behavior? do we know why we had the old behavior in the first place?

Reply:
That looks reasonable to me that the tokens focusing behaviour is consistent no matter where the token group is used. However, I'll check if we do want a different treatment for multiselect.

Reply:
so long as someone on the design team has seen it it should be fine

Reply:
UX approved :shipit: